src/main/scala/mesosphere/marathon/api/akkahttp/EntityMarshallers.scala

```diff
@@ -163,6 +177,7 @@ object EntityMarshallers {
   implicit val enrichedTaskMarshaller = playJsonMarshaller[raml.EnrichedTask]
   implicit val enrichedTasksListMarshaller = playJsonMarshaller[raml.EnrichedTasksList]
   implicit val deleteTasksUnmarshaller = playJsonUnmarshaller[raml.DeleteTasks]
+  implicit val seqDateTimeMarshalled = playJsonMarshaller[Seq[Timestamp]]
```

Comment:
Interesting. So it can't figure out how to serialize `Seq[T]` given a marshaller for type `T`?

Reply:
apparently no :( any idea where is the problem? If I remove it, this is the error

found   : scala.collection.immutable.Seq[mesosphere.marathon.state.Timestamp]
[error]  required: akka.http.scaladsl.marshalling.ToResponseMarshallable
[error]             complete(versions)